CRM/Activity/BAO/Activity.php

```diff
@@ -2107,6 +2107,7 @@ public static function addActivity(
     $id = CRM_Core_Session::getLoggedInContactID();
     if ($id) {
       $activityParams['source_contact_id'] = $id;
+      $activityParams['target_contact_id'][] = $activity->contact_id;
```

Comment:
This is required when registering an additional participant and activity gets created for only primary contact. As we get the logged in contact id we replace the source_contact with it. Keeping the `$activity->contact_id` in target would show the activities for additional participant too.

shall I check for !empty($activity->contact_id) in case this might not populate but I see this has been already [assigned to source_contact](https://github.com/civicrm/civicrm-core/blob/master/CRM/Activity/BAO/Activity.php#L2088) which is being replaced here, so it should always be there?